notebook_index.yml

```diff
@@ -119,6 +119,8 @@ contents:
             description: Detailed examples on importing and implementing IDAES reactor models in flowsheets
             notebooks:
               - cstr: CSTR unit model with ideal property package, pure liquid phase and rate-kinetics reaction
+              - plug_flow_reactor: Plug flow reactor unit model with ideal property package, pure liquid phase and rate-kinetics reaction
```

Comment:
Not sure what is meant by "pure" liquid phase here. Do you mean "single"?

Also line below.

Reply:
The three rate-reactor examples exhibit liquid phase behavior only (no vapor).